README.md

```diff
@@ -11,4 +11,34 @@ the folder 3rdparty.
 
 Screenshots
 -----------
-![Alan Wakes American Nightmare 1](screenshots/awan1.png)
\ No newline at end of file
+![Alan Wakes American Nightmare 1](screenshots/awan1.png)
+
+
+Building and running on Linux
+--------
+
+Make sure to not include the `$` when running these. It signifies that the commands should be run as your user, not root.
```

Comment:
Hmm, I think it is unnecessary to point people to not use root when compiling this, that should hopefully be clear to anyone, and since I cannot think of any component of OpenAWE, which needs root access, it shouldn't also be needed for differentiation.

Reply:
Usually `$` is used to denote a shell command which is what I'm trying to explain here. I think it's useful to include to make it clear where one command ends and another starts. But yes, this comment could explain that better.

That said, do you prefer if I remove the `$`'s altogether?

Reply:
Ok, after thinking about it, I think we keep it with the $ chars